typings/index.d.ts

```diff
@@ -14,10 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export type ErrorEvent = {
-  message: string,
-  filename: string,
-  lineno: number,
-  colno?: number,
-  error?: any,
-};
+/// <reference path="custom.d.ts" />
```

Comment:
Does this file serve a purpose if it is just comments?

Reply:
Hey, thought I responded to this. Yes, it does. Added docs for this, here:

https://github.com/jaegertracing/jaeger-ui/pull/331/files#diff-c21202b241d0cd40822db520c8b00770R103